HomeworkBasic1.ipynb

```diff
@@ -0,0 +1,192 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assignment 1\n",
+    "\n",
+    "Calculate the sum of all the squared numbers from 1 to 100. Keep in mind that the `range()` function excludes the given upper limit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[0, 1, 4, 9, 16, 25, 36, 49, 64, 81, 100]\n",
+      "385\n"
+     ]
+    }
+   ],
+   "source": [
+    "import math\n",
+    "squares = [x**2 for x in range(11)]\n",
+    "print (squares)\n",
+    "total = sum(squares)\n",
+    "print(total)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assignment 2\n",
+    "\n",
+    "Calculate the sum of all the numbers in an array. There are multiple ways of doing this, choose any you like."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "45\n"
+     ]
+    }
+   ],
+   "source": [
+    "a = [0,1,2,3,4,5,6,7,8,9]\n",
+    "b = sum(a)\n",
+    "print (b)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assignment 3\n",
+    "\n",
+    "Adding to assignment 2, implement the same functionality but allow the array to be a mix of numbers and numbers represented as strings, e.g. [1, 5, '7', 5, '10']."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "224\n"
+     ]
+    }
+   ],
+   "source": [
+    "num1 = 100\n",
+    "num2 = 10\n",
+    "num3 = 50\n",
+    "a = [num1, 5, num2, 10, num3, 49]\n",
```

Comment:
this is still an array of numbers, not of numbers and strings. You just put some numbers into variables.